res/houdini/MainMenuCommon.XML

```diff
@@ -63,7 +63,7 @@ publish.show(parent=pipeline.get_main_window())
                 <label>Work Files ..</label>
                 <scriptCode><![CDATA[
 import hou
-from avalon.tools import workfiles
+from pype.tools import workfiles
```

Comment:
we don't need to do it here for houdini. We're overwriting this file completely in Pype anyway